test/modules/aws-account-integration.ts

```diff
@@ -121,13 +129,38 @@ describe('AwsAccount Integration Testing', () => {
     })();
   });
 
-  it('inserts a second, useless row into the aws_account table', query(`
-    INSERT INTO aws_account (access_key_id, secret_access_key, region)
-    VALUES ('fake', 'creds', 'us-west-2')
+  it('inserts a second, useless row into the aws_credentials table', query(`
+    INSERT INTO aws_credentials (access_key_id, secret_access_key) VALUES ('fake', 'creds')
   `));
 
   it('does absolutely nothing when you apply this', apply());
 
+  it('selects a default region', query(`
+    UPDATE aws_regions SET is_default = TRUE WHERE region = '${process.env.AWS_REGION}';
+  `));
+
+  it('confirms that the default region was set', query(`
+    SELECT * FROM aws_regions WHERE is_default = TRUE;
+  `, (res: any[]) => expect(res.length).toBe(1)));
+
+  it('tries to set a second default region', query(`
+    UPDATE aws_regions SET is_default = TRUE WHERE region = 'us_east_1';
+  `));
+
+  it('confirms that the default region was not changed', query(`
+    SELECT * FROM aws_regions WHERE is_default = TRUE;
+  `, (res: any[]) => {
+    expect(res.length).toBe(1);
+    expect(res[0].region).toBe(process.env.AWS_REGION);
+  }));
```

Comment:
so we cannot have 2 default regions but we could have the case where none of the regions is the default one, right? In that case the postgres fn will return `us-east-1`